sql-odbc/docs/user/power_bi_support.md

```diff
@@ -8,39 +8,33 @@
 * Optional: [odfesqlodbc_import.pbids](../../src/PowerBIConnector/PBIDSExamples) to help with repeated connections to the same server 
 
 ## Setup
-* Copy `OdfeSqlOdbcPBIConnector.mez` file in the `<User>\Documents\Power BI Desktop\Custom Connectors\` folder. This will let Power BI access custom connector.
-* Open Power BI Desktop.
-* Change the security settings. Click on **Files** > **Options and settings** > **Options** > **Security** > Select **Allow any extension to load without validation or warning** for Data Extensions. This will allow the custom connector to load data into Power BI.
-
-<img src="img/pbi_settings.png" width="500">
-
-* Restart Power BI Desktop.
+* Copy `OdfeSqlOdbcPBIConnector.mez` file in the `<User>\Documents\Power BI Desktop\Custom Connectors\` folder. This will let Power BI access custom connector. If you have installed the On-premises data gateway, you should be able to see the ODBC is detected as a custom connector in the **Connectors** tab.
```

Comment:
1. Copy `OdfeSqlOdbcPBIConnector.mez` file to
2. Could we use the msi to install the mez?
3. Does the mez need the release procedure, how do we maintain it in furutre?

Reply:
Corrected the typo. For the mez file, we need more discussion over it offline, let's keep it out of scope for this pr.